### ISSUE TITLE
Accessibility of courses based on course instance start/end times

### ISSUE THREAD

**Issue by vkaravir** [Aalto-LeTech/a-plus#52]
The course instance accessibility based on start/end times should be tweaked a bit:

 * A course instance where start and end times are in the past should not be shown on the front page. It should be shown in the archive and should be accessible by the students and staff on that course.
 * A course instance started should be shown on front page and accessible by everyone.
 * A course instance not yet started should be shown on front page but accessible only by superusers and teachers and assistants of that course instance.

**Comment by teemulehtinen** [comment]
Implemented.
